pkg/controller/manager/manager_controller.go

```diff
@@ -327,15 +328,24 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		return reconcile.Result{}, err
 	}
 
-	trustedSecretNames := []string{render.PacketCaptureCertSecret, monitor.PrometheusTLSSecretName, relasticsearch.PublicCertSecret, render.ProjectCalicoApiServerTLSSecretName(installation.Variant)}
-
+	// Determine if compliance is enabled.
 	complianceLicenseFeatureActive := utils.IsFeatureActive(license, common.ComplianceFeature)
 	complianceCR, err := compliance.GetCompliance(ctx, r.client)
 	if err != nil && !errors.IsNotFound(err) {
 		r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying compliance: ", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 
+	// Build a trusted bundle containing all of the certificates of components that communicate with the manager pod.
+	// This bundle contains the root CA used to sign all operator-generated certificates, as well as the explicitly named
+	// certificates, in case the user has provided their own cert in lieu of the default certificate.
+	trustedSecretNames := []string{
+		render.PacketCaptureCertSecret,
+		monitor.PrometheusTLSSecretName,
+		relasticsearch.PublicCertSecret,
+		render.ProjectCalicoAPIServerTLSSecretName(installation.Variant),
```

Comment:
I'm not sure that the API server TLS secret is actually needed here, but it was pre-existing so leaving it for now.

Nothing should be talking directly to the Calico API server, AFAIK

Reply:
The apiserver deploy also has a queryserver. Both mount the same TLS secret. Maybe that is why?

Reply:
Hm, yeah maybe that's it.